assets/js/main.js

```diff
@@ -370,4 +370,58 @@ $(function() {
   };
   anchors.add().remove('.no-anchor');
   anchors.add('body.terms .terms-row .tos-table__legal p strong');
-});
\ No newline at end of file
+});
+
+/* Add cookie prompt functionality */
+(function(){
+  var linkText = "cookie policy";
+  var linkUrl = "/cookie-policy/";
+  var dismissBtn = "OK";
+  var msg = "By using this website you agree to our " + linkText.link(linkUrl);
+
+
+  //check if cookie already exists
+  if(document.cookie){
```

Comment:
Might be cleaner to add a reusable `getCookie(name)` function: https://stackoverflow.com/a/25490531/483528